Documentation/cmdref/cilium_service_update.md

```diff
@@ -11,6 +11,7 @@ cilium service update [flags]
 ### Options
 
 ```
+      --backend-weights uints       Backend weights (1 default, 0 means maintenance state) (default [])
```

Comment:
We should mention that weights work only for Maglev.

Reply:
sg